main/appserver/tests/rq.sh

```diff
@@ -205,7 +205,7 @@ then
     echo -e $USAGE
     exit 0
 fi    
-while getopts ":b:t:g:a:l" opt; do
+while getopts ":b:t:g:al" opt; do
```

Comment:
I just realized the usage text at line 176 needs to be fixed, can you please do that?